Web/starter_project_barbies/components/common/TagList.tsx

```diff
@@ -5,10 +5,7 @@ interface TagListProps {
 }
 
 export const TagList: React.FC<TagListProps> = ({ tags }) => {
-  let tagComponents = []
-  for (let tag of tags) {
-    tagComponents.push(<Tag tag={tag} />)
-  }
+  let tagComponents = tags.map((tag, index) => <Tag tag={tag} key={index} />)
```

Comment:
You don't need to store it in a vairable, it can be moved inside the div on line 12 as it is